pkg/dashboard/index.html

```diff
@@ -84,9 +84,9 @@
                 <div class="panel panel-default" id="dashboard-hosts">
                     <div class="panel-heading">
                         <button class="btn btn-primary servers-privileged fa fa-plus"
-                            id="dashboard-add" title="Add Server"></button>
+                            id="dashboard-add" aria-label="Add Server" title="Add Server"></button>
                         <button class="btn btn-default fa fa-check"
-                            id="dashboard-enable-edit" title="Edit Server"></button>
+                            id="dashboard-enable-edit" aria-label="Edit Server" title="Edit Server"></button>
```

Comment:
Same issue as above. It's untranslatable as-is.

Note: If these buttons contained text, we could mark it as translatable with `translate` and drop the `title` and `aria-label`s.